crates/pallet-domains/src/tests.rs

```diff
@@ -20,6 +20,9 @@ type Balance = u64;
 // TODO: Remove when DomainRegistry is usable.
 const DOMAIN_ID: DomainId = DomainId::new(0);
 
+// Operator id used for testing
+const OPERATOR_ID: OperatorId = 0u64;
```

Comment:
It is not the case here IMO, there is only one test using it because its value doesn't matter and just serves as a placeholder, even if we define it inside the test, the value will still be the same thus doesn't bring any help on debugging.